recipes/fish/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+export CFLAGS="-I$PREFIX/include -I$PREFIX/include/ncursesw"
+export CXXFLAGS="-I$PREFIX/include -I$PREFIX/include/ncursesw -static-libstdc++"
+
+autoconf
+./configure --prefix=$PREFIX
+make -j4
```

Comment:
Assuming 4 here may be problematic. You should change to the env var CPU_COUNT instead.